exotica/CMakeLists.txt

```diff
@@ -51,20 +51,8 @@ include(GetGitRevisionDescription)
 get_git_head_revision(GIT_REFSPEC GIT_SHA1)
 configure_file("src/Version.cpp.in" "${exotica_BINARY_DIR}/generated/Version.cpp" @ONLY)
 
-AddInitializer(
-  TaskMap
-  Object
-  Limb
-  MotionSolver
-  PlanningProblem
-  Scene
-  TaskDefinition
-  UnconstrainedTimeIndexedProblem
-  UnconstrainedEndPoseProblem
-  SamplingProblem
-  TaskSqrError
-  TaskTerminationCriterion
-)
+AddInitializer(TaskMap Object Frame MotionSolver PlanningProblem Scene
```

Comment:
Why this formatting change? It's easier to add and remove with the open array without changing the whole list of initializers